===================================================================
RCS file: /cvs/gcc/gcc/gcc/config/rs6000/rs6000.md,v
retrieving revision 1.284.4.11
retrieving revision 1.284.4.12
diff -u -r1.284.4.11 -r1.284.4.12
--- gcc/gcc/config/rs6000/rs6000.md	2004/07/15 21:03:10	1.284.4.11
+++ gcc/gcc/config/rs6000/rs6000.md	2004/07/27 01:51:40	1.284.4.12
@@ -8549,7 +8549,7 @@
 }")
 
 (define_insn "*movdi_internal64"
-  [(set (match_operand:DI 0 "nonimmediate_operand" "=Y,r,b,r,r,r,r,r,?f,f,m,r,*h,*h")
+  [(set (match_operand:DI 0 "nonimmediate_operand" "=Y,r,b,r,r,r,r,r,??f,f,m,r,*h,*h")
 	(match_operand:DI 1 "input_operand" "r,Y,m,r,I,L,nF,R,f,m,f,*h,r,0"))]
   "TARGET_POWERPC64
    && (gpc_reg_operand (operands[0], DImode)
